Vagrantfile

```diff
@@ -19,8 +19,8 @@ CONF = _config
 MOUNT_POINT = '/home/vagrant/kitsune'
 
 Vagrant::Config.run do |config|
-    config.vm.box = "ubuntu-14.04"
-    config.vm.box_url = "https://oss-binaries.phusionpassenger.com/vagrant/boxes/latest/ubuntu-14.04-amd64-vbox.box"
+    config.vm.box = "ubuntu/trusty32"
+    config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty32/versions/14.04/providers/virtualbox.box"
```

Comment:
I think the default should be a 64-bit operating system since that's closer to what we're using in production. If you feel strongly about supporting 32-bit operating systems, then you can do some finagling in the `vagrantfile.yml` file to support people overriding the default.